modules/RTC/RTCUtils.js

```diff
@@ -869,17 +869,41 @@ class RTCUtils extends Listenable {
                 this.p2pPcConstraints
                     = JSON.parse(JSON.stringify(this.pcConstraints));
 
-                // Allows sending of video to be suspended if the bandwidth
-                // estimation is too low.
-                if (!options.disableSuspendVideo) {
-                    this.pcConstraints.optional.push(
-                        { googSuspendBelowMinBitrate: true });
+                const {
+                    forceSuspendVideo,
+                    disableSuspendVideo
+                } = options;
+                const suspendVideoEnabled
+                    = typeof forceSuspendVideo === 'undefined'
+                        || (typeof forceSuspendVideo === 'number'
+                            && Math.random() < forceSuspendVideo);
```

Comment:
looks like `forceSuspendVideo` might be better named as `forceSuspendVideoProbability`?

Reply:
and am i reading this right that, if `forceSuspendVideo` isn't defined at all (typeof === undefined), we equate that to '100% probability'?

Reply:
whatever default we want for `forceSuspendVideo`, maybe we could do it once at the top like: `forceSuspendVideo = forceSuspendVideo || 1` or something (since we have to check it isn't undefined in multiple places)

Reply:
Thanks for the proposal! 

I'm usually very bad with naming variables, etc. I'll change the name! 

Also I like the idea about `forceSuspendVideo = forceSuspendVideo || 1`!